src/routes/devices/stats.ts

```diff
@@ -1,16 +1,19 @@
+import { DeviceDb } from '../../db/device'
+
 /*********************************************************************
  * Copyright (c) Intel Corporation 2021
  * SPDX-License-Identifier: Apache-2.0
  **********************************************************************/
 export async function stats (req, res): Promise<void> {
-  const amtCredentials = await req.mpsService.db.getAllAmtCredentials()
+  const db = new DeviceDb()
+  const devices = await db.get()
   let connectedCount = 0
   let totalCount = 0
-  if (amtCredentials != null) {
-    totalCount = Object.keys(amtCredentials).length
+  if (devices != null) {
+    totalCount = Object.keys(devices).length
```

Comment:
I don't think this is correct. Devices is an array now right? Should be just .length without need for object keys

Reply:
done